tools/Gemfile.plugins.test

```diff
@@ -14,3 +18,10 @@ plugins = [ 'logstash-filter-clone',
 plugins.each do |plugin|
   gem plugin
 end
+
+##
+# Dependencies related with coverage analysis
+##
+
+gem 'simplecov'
+gem 'coveralls'
```

Comment:
Let me check, I declared them here because there was a comment raising an issue with bundler and development dependencies.